gnome-image-installer/gnome-image-installer.c

```diff
@@ -365,14 +361,6 @@ gis_page_get_type();
 #endif
 
   gtk_init (&argc, &argv);
-  ev_init ();
-
-#if HAVE_CLUTTER
-  if (gtk_clutter_init (NULL, NULL) != CLUTTER_INIT_SUCCESS) {
-    g_critical ("Clutter-GTK init failed");
-    exit (1);
-  }
-#endif
```

Comment:
Why remove these? Deprecated? Unused? Could do with a comment in the commit message.